core/assets/static/images/icons/signout_active.svg

```diff
@@ -0,0 +1,88 @@
+%PDF-1.7
```

Comment:
This seems to be PDF, so not an svg (like the filename implies). Is there a reason for this?

Reply:
Oops.. will change it